packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -723,6 +763,21 @@ export function getMetaContent(metaName: string): string | undefined {
   return metaTag?.getAttribute('content') || undefined;
 }
 
+/**
+ * Manually report the end of the page load, resulting in the SDK ending the pageload span.
+ * This only works if {@link BrowserTracingOptions.explicitPageloadEnd} is set to `true`.
+ * Otherwise, the pageload span will end itself based on the {@link BrowserTracingOptions.finalTimeout},
+ * {@link BrowserTracingOptions.idleTimeout} and {@link BrowserTracingOptions.childSpanTimeout}.
+ *
+ * @param client - The client to use. If not provided, the global client will be used.
+ */
+export function reportPageLoaded(client?: Client): void {
```

Comment:
```suggestion
export function reportPageLoaded(client?: Client = getClient()): void {
```

I think this should also work, then you can remove the `clientToUse` stuff :)